stanford_person.install

```diff
@@ -136,3 +136,55 @@ function stanford_person_update_7500(&$sandbox) {
 
 
 }
+
+/**
+ * Implements hook_install().
+ */
+function stanford_person_install() {
+  stanford_person_set_ds_region_blocks();
+}
+
+/**
+ * Implements hook_uninstall().
+ */
+function stanford_person_uninstall() {
+
+  // Remove the DS region blocks variables.
+  $settings = variable_get('ds_extras_region_blocks');
+  unset($settings['s_news_image']);
+  unset($settings['office_hours']);
+  unset($settings['contact']);
+  unset($settings['research_interests']);
+  variable_set('ds_extras_region_blocks', $settings);
+}
+
+/**
+ * Set the DS region block values.
+ */
+function stanford_person_set_ds_region_blocks() {
+
+  // Add the DS region blocks variables.
+  $settings = variable_get('ds_extras_region_blocks');
```

Comment:
variable_get('ds_extras_region_blocks', array());

And you probably want to check to ensure that the settings don't already exist as to ensure you don't overwrite something by accident.